test/functional/ex_cmds/encoding_spec.lua

```diff
@@ -19,22 +19,20 @@ describe('&encoding', function()
     eq('utf-8', eval('&encoding'))
     -- check nvim is still in utf-8 mode
     eq(3, eval('strwidth("Bär")'))
+
+    clear('set enc=latin1')
+    execute('set encoding=utf-8')
+    feed('<cr>')
+    eq('latin1', eval('&encoding'))
+    eq(4, eval('strwidth("Bär")'))
```

Comment:
How about moving this to a separate test, `it('can be changed before startup')`.